FirebaseAppCheck/Apps/FIRAppCheckTestApp/FIRAppCheckTestApp/AppDelegate.swift

```diff
@@ -24,8 +24,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   func application(_ application: UIApplication,
                    didFinishLaunchingWithOptions launchOptions: [UIApplication
                      .LaunchOptionsKey: Any]?) -> Bool {
+    let providerFactory = AppCheckDebugProviderFactory()
+    AppCheck.setAppCheckProviderFactory(providerFactory)
```

Comment:
Can you move these lines inside the `requestLimitedUseToken` method? Since the other `request` test methods here install their own App Check providers.

Reply:
Can I call Firebase.configure() again in this method?

Reply:
Ah hmm not sure -- @ncooke3 does `FirebaseApp.configure()` have to be called after `setAppCheckProviderFactory`?

Reply:
Thx for confirming, I will just keep it for now